vonk/features/validation.rst

```diff
@@ -78,5 +78,6 @@ Limitation
 -----------
 
 #. The mode parameter is not yet supported.
+#. Implicit ValueSets from a CodeSystem are not supported - create explicit ones instead.
```

Comment:
I think this requires an example or link to an explanation in the spec for most people.

Reply:
I've clarified this.